### PR TITLE
CI: Improve spectests coverage report

### PR DESCRIPTION
**Pull request by chfast**
See #644.

### CODE REVIEW

**Comment by chfast** [comment]
I can squash into single commit.

File: circle.yml [hunk 1]
```diff
@@ -138,11 +138,11 @@ commands:
           working_directory: ~/build
           command: ctest -E 'unittests|smoketests' -j4 --schedule-random --output-on-failure
 
-  collect_coverage_data:
-    description: "Collect coverage data"
+  collect_coverage_clang:
```
Comment (axic) [inline circle.yml:141]: Commit message says llvm, this here is clang, the text below is llvm. Choose one :)

File: circle.yml [hunk 2]
```diff
@@ -145,17 +145,21 @@ commands:
           name: "Collect coverage data (LLVM)"
           working_directory: ~/build
           command: |
-            binaries='-object bin/fizzy-unittests -object bin/fizzy-spectests -object bin/fizzy-bench -object bin/fizzy-testfloat -object bin/fizzy-wasi'
+            OBJECTS='fizzy-bench fizzy-spectests fizzy-testfloat fizzy-unittests fizzy-wasi'
+
+            for OBJECT in $OBJECTS; do
+              OBJECTS_ARGS="$OBJECTS_ARGS -object bin/$OBJECT"
```
Comment (axic) [inline circle.yml:155]: Nitpick: I think `OBJECT_ARGS` or `OBJECTS_ARG` is better. But good either way.

**Review by axic — APPROVED** [review]
Looks good in essence, but there are some nitpicking comments.

**Comment by chfast** [comment]
> I can squash into single commit.

Squash or not?

File: circle.yml (hunk 1 above)
Comment (chfast) [reply circle.yml:141]: Although LLVM is formally more correct, I'm sticking with the Clang name.

**Comment by axic** [comment]
> > I can squash into single commit.
> 
> Squash or not?

Up to you, if you squash then leave some explainer in the commit log.

**Pull request merged**